MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/utils/configuration/ConfigUtils.java

```diff
@@ -93,4 +93,20 @@ public static void saveConfig(JavaPlugin plugin, FileConfiguration config, Strin
     }
   }
 
+  /**
+   * Sets the data into the config file if it is absent
+   *
+   * @param config FileConfiguration to use
+   * @param key    Key of the data value to set
+   * @param data   The data to be set
+   *
+   * @return       Return value is true if the data value was not set and has now been set.
+   */
+  public static boolean setIfAbsent(FileConfiguration config, String key, Object data) {
+    if (!config.isSet(key) || !config.contains(key) || config.get(key) == null) {
```

Comment:
These 3 condition literally doing the same check.

Reply:
Redundancy in case one of them breaks for some random reason.

Reply:
I doubt it, using the `get` method is completely sufficient. If you want a deep check, you can use the `isSet` method as this checks to copy the default value or not.

Reply:
why we got this if its never used?

Reply:
oh, I didn't even see it isn't used